### PR TITLE
Add several v1 routes

### PR DESCRIPTION
**Pull request by IAmVisco**

### CODE REVIEW

File: apps/client-api/routes/v1/channels.js [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
+
+router.get('/:channel_id', asyncMiddleware(async (req, res) => {
+  const { channel_id } = req.params;
 
-  // Run through all results
-  channels.map(video => {
-    const channelData = video.data();
-    const channelObj = new HoloChannel(channelData);
-    return results.channels.push(channelObj.toJSON());
+  const key = parseInt(channel_id, 10) ? 'bb_space_id' : 'yt_channel_id';
```
Comment (sphinxrave) [inline apps/client-api/routes/v1/channels.js:34]: lol...

Comment (sphinxrave) [reply apps/client-api/routes/v1/channels.js:34]: I think channelId should be db `CHANNEL.id` not a arbitrary Identifier.

File: apps/client-api/routes/v1/videos.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  } = req.query;
+
+  const where = {
+    ...title && { title: { [Op.iLike]: `%${title}%` } },
+    // TODO: Figure out and fix timezones
+    ...published_at && { published_at: { [Op.gte]: moment(published_at).startOf('day') } },
+    ...status && { status },
+    // Hacky way to convert string booleans into real booleans
+    ...is_uploaded && { is_uploaded: JSON.parse(is_uploaded.toLowerCase()) },
+    ...is_captioned && { is_captioned: JSON.parse(is_captioned.toLowerCase()) },
```
Comment (dragonjet) [inline apps/client-api/routes/v1/videos.js:31]: we can consider defining valid request values as 1 or 0

File: apps/client-api/routes/v1/videos.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    published_at,
+    status,
+    is_uploaded,
+    is_captioned,
+  } = req.query;
+
+  const where = {
+    ...title && { title: { [Op.iLike]: `%${title}%` } },
+    // TODO: Figure out and fix timezones
+    ...published_at && { published_at: { [Op.gte]: moment(published_at).startOf('day') } },
```
Comment (dragonjet) [inline apps/client-api/routes/v1/videos.js:27]: can we have an "until" filter? if let's say I wanna fetch only one specific day

File: apps/client-api/routes/v1/videos.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    include: [
+      {
+        association: 'channel',
+        attributes: RESPONSE_FIELDS.CHANNEL,
+      },
+    ],
+    where: {
+      [Op.or]: {
+        yt_video_key: video_id,
+        bb_video_id: video_id,
```
Comment (dragonjet) [inline apps/client-api/routes/v1/videos.js:71]: same with @sphinxrave's comment on the channels endpoint, we can treat video_id as our database ID in `GET /videos/1`. then the `yt_video_key` and `bb_video_id` can just be part of filters in the `GET /` (e.g. `?yt_video_key=`).

File: apps/client-api/routes/v1/videos.js (hunk 2 above)
Comment (IAmVisco) [reply apps/client-api/routes/v1/videos.js:31]: They are still going to be strings when parsed from query string so our course of action going to be `is_captioned: !!+is_captioned`

**Pull request merged**